tests/test_management.py

```diff
@@ -22,6 +20,8 @@
 from django.utils import timezone
 
 from enterprise import lms_api
+from integrated_channels.integrated_channel.learner_data import BaseLearnerExporter
```

Comment:
@snowcrshd Hint: `isort` with `--check-only` checks that imports are sorted as it see fit (there's config somewhere, maybe in tox.ini). Without `--check-only` it'll actually sort imports itself.

But I'm a bit puzzled why this change is needed in the first place - isort didn't complain here previously (AFAIK).